data/formatters/generic.yaml

```diff
@@ -149,6 +149,17 @@ short_source: 'LOG'
 source: 'GoogleDriveSync Log File'
 ---
 type: 'conditional'
+data_type: 'googlelog:log'
+message:
+  - '{file_name}:'
+  - '{line_number}]'
```

Comment:
There is a `]` in this message part but no `[` is that intended? Do you also want add context what these values represent e.g. `Line number: {line_number}` ?

Reply:
IMHO not required, it is now user configurable, people can change it if they don't like the defaults. But might be worth a follow up conversation about some unification here like the event data attributes.